src/v8/isolate_scope.rs

```diff
@@ -28,15 +28,77 @@ use crate::v8::v8_set::V8LocalSet;
 use crate::v8::v8_string::V8LocalString;
 use crate::v8::v8_unlocker::V8Unlocker;
 use crate::v8::v8_value::V8LocalValue;
+use crate::v8_c_raw::bindings::v8_isolate;
 
 use std::os::raw::{c_char, c_void};
 
-pub struct V8IsolateScope<'isolate> {
-    pub(crate) isolate: &'isolate V8Isolate,
+/// An owned, unpinned storage of the handlers and isolate scopes.
```

Comment:
What do you mean by storage here?

Reply:
Its a very dangerous start, it holds direct pointers to object he is not the owner of and it does not have a lifetime.

Reply:
Maybe it is the owner of 
```
inner_handlers_scope: *mut v8_handlers_scope,
    inner_isolate_scope: *mut v8_isolate_scope,
```
But its not the owner of `original_isolate: *mut v8_isolate,`